langchain-core/src/runnables/base.ts

```diff
@@ -550,6 +551,22 @@ export abstract class Runnable<
     );
   }
 
+  getGraph(_?: RunnableConfig): Graph {
+    const graph = new Graph();
+
+    // TODO: what is the inputNode here?
+    const inputNode = graph.addNode({} as unknown as Runnable);
```

Comment:
@nfcampos can I get some pointers on how to solve this? seems like the `get_input_schema` isn't implemented yet in JS, unlike [in python](https://github.com/langchain-ai/langchain/blob/a99eb3abf4db7ce98fb271df93c23867583782b8/libs/core/langchain_core/runnables/base.py#L393-L394)

Reply:
I think it's fine to leave this for now - we can add at a later date when we add support for input/output schemas.